dragonfly/world/loader.go

```diff
@@ -124,6 +126,13 @@ func (l *Loader) reset() {
 		l.w.removeViewer(pos, l.viewer)
 	}
 	l.loaded = map[ChunkPos]*chunk.Chunk{}
+}
+
+// newWorld sets the loader's world, adds them to the world's viewer list, then starts populating the load queue.
+// This is only here to get rid of duplicated code, ChangeWorld should be used instead of this.
+func (l *Loader) newWorld(new *World) {
```

Comment:
Naming of this is a bit awkward, might be better to call it `setWorld` or just `world`.